src/ui/LoadingScreen.tsx

```diff
@@ -34,8 +34,9 @@ export function LoadingScreen(): React.ReactElement {
   });
 
   useEffect(() => {
-    load()
-      .then((saveData) => Promise.all([initSwc(), Engine.load(saveData)]))
+    initSwc()
```

Comment:
initSwc just needs to happen before `Engine.load`, not `load()`, right?
To save some time on init (hopefully) can we `Promise.all([initSwc, load()]).then(Engine.load())`?